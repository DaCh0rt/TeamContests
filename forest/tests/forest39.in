698 823
-1900 695
8027 -4222
1316 -5145
-1355 863
7603 -524
1173 7776
7633 3938
-518 -8386
-2548 8151
-7287 -138
2873 -8700
-8871 -2044
4768 3832
-5849 -704
618 -2058
-9092 -8470
-319 -9744
3445 -2043
3484 -7752
-377 4572
-1439 9056
229 3762
1539 -3093
6428 -154
-6718 1377
-6535 8661
2761 -2181
-531 4390
-4502 -3694
8218 3937
3701 3421
-4230 2303
-7279 5731
1363 9382
942 8005
9418 -5053
8150 -720
3376 9337
-7948 6782
-5308 8242
7025 925
4129 1118
-9608 9821
-7156 3133
2517 -8587
-4007 -6953
93 -3271
8222 -1778
-3836 -2705
-927 -352
-6815 7945
7283 5204
1875 2964
-2045 567
7096 9624
952 -2823
-1102 -8969
-64 -1434
-6089 -7761
9450 -7062
4333 -3574
-9218 6445
8103 3138
-5387 7929
5157 4048
647 4031
-6429 -8157
7028 -110
7313 -2625
9842 -5686
6612 -1580
830 9502
4390 -6954
-9231 -6335
1125 9211
2062 8636
-1057 3767
7517 -5423
-834 -7431
-9554 4340
736 -4479
7187 812
-2095 448
-2910 9180
-5704 475
1269 -5335
-8259 -7965
-1622 2612
8123 2179
-6972 -284
8175 8123
-5550 -2391
-8416 -9218
-2542 -5980
-9686 -3571
-3587 -3701
-7148 -3351
2023 -2114
2957 -2430
6745 150
-4152 9282
8812 8216
-1831 -3701
-8592 -2594
6888 -3563
6458 5445
-2565 -7953
6133 2184
-7800 -8323
3159 -2676
-5875 210
9831 9015
2254 4047
-9544 7697
-7739 -1104
-4569 -3755
-7342 -8453
1180 2989
-9476 -5841
7138 -7116
-9386 4155
-2433 9033
-2889 7505
-5013 3873
554 5872
-4912 -6161
-452 1461
6610 -5283
858 -313
-7629 -7266
-7548 8861
-1199 7958
5609 -753
-2906 -4574
1887 -2063
8425 9407
7967 -5830
-3485 379
-6322 -2174
4160 -6199
4412 -1385
1767 -9358
-6487 -125
6984 7144
-3834 -9420
2378 -1615
2063 -1348
9543 -3769
-8164 4014
-7630 -6495
309 -1674
-5851 -452
-63 8242
1244 3405
3536 -4761
2451 9213
7325 1640
8262 -3275
-9570 -4482
4950 -3340
4716 1735
9897 5577
4147 452
-139 5617
2180 225
3165 -7839
-1730 -1577
9526 -1604
-7909 -9048
1277 -5463
-7349 -1742
186 6902
4391 6599
8485 -3922
3268 -5039
1658 -966
-7372 -1587
-727 3855
-4882 -6998
7527 -3819
-9776 2372
8837 -6380
3680 -1153
2912 -3872
-2185 2595
6937 -3256
-2553 -3592
6619 -5585
3882 -6049
952 3142
9438 -4354
7574 -8486
-4771 9485
5708 5160
-7190 8840
-1975 -485
4024 -9607
8845 -5604
-7545 -1008
6739 4086
6650 5325
-9104 7522
5998 1187
8305 3712
1347 -610
-1996 7435
227 5355
-447 1555
-3625 4954
9535 726
-2874 638
933 -6686
2144 -950
4772 417
-8148 -8796
-2118 -5514
-3931 737
6667 4214
-4134 1656
-5534 -3697
-9955 -9259
-9843 634
-7869 -8954
-4817 -7374
-1760 -4277
1061 -3461
4361 -5085
2553 -7960
5896 5605
5724 3683
-3988 2569
2477 -9245
5403 5777
5131 5518
6135 -1012
-9048 1772
-5141 -3150
3 473
-605 1463
-1880 2863
6529 -7124
-6774 8976
-4668 1794
-2156 -4593
-2682 558
-1807 -4091
4775 8704
7432 -8074
6613 1587
1862 8973
-1708 -6537
2370 -6523
-2598 6177
4632 1584
-4784 -7343
9059 1023
-6714 -8674
2523 3053
-9636 194
462 -2735
6053 7332
9817 8373
5882 2694
2534 7928
5754 6984
4368 -9961
9811 2209
-3546 6956
4886 -4500
-352 6903
5146 -4483
-19 -8255
7853 2231
-6549 814
71 1572
9051 -2002
6836 1084
7629 9554
9313 7899
-6163 25
5211 -1203
6837 7574
797 7318
-6863 6525
6053 6110
5761 7331
1689 -7842
1995 -7956
-5381 -2500
3498 -1407
1801 -7360
7869 -9731
-2114 5855
-3714 -6903
589 -1156
-4259 -7590
-7525 -9342
4787 2720
9341 1484
-7595 6067
-5023 8301
2551 -5624
5056 5876
-3187 -9739
7480 9697
-3680 -2209
9671 -3704
-3304 4624
385 -9092
6105 -8320
2376 518
7308 1640
-2401 -8663
8161 -452
-1721 -2789
278 7621
2617 -9166
-7753 3907
-2850 -4240
-1931 -3689
-1283 -4889
9913 -9054
-4811 -1314
-2346 -9812
7940 -131
2335 -9743
-7352 8220
2142 2536
9558 -5461
-4279 3905
-1921 -162
-6861 -5012
-9181 9243
-7473 1675
-9059 -2233
8288 8159
9389 7984
9798 3874
-9539 -7652
-5230 8522
-3409 -3508
6156 6435
9678 -5766
-2532 3056
4790 5970
-3732 9929
-7420 2519
4982 -7052
8304 -3067
1217 -2777
7010 1309
-3199 -8489
8041 5335
8195 -4750
-1577 -4425
7331 6249
6858 5919
5946 -8241
-9943 -9837
-9188 -7119
1604 2978
609 4468
-6372 -8969
-680 8577
4415 -3706
-8886 -659
-1986 522
9622 6149
-7351 -705
1841 2817
-3739 -697
-4780 2987
7323 5494
-3278 3560
-9914 8665
-4964 -7784
3356 1830
-6881 -7608
6478 6728
7845 -451
-3077 -1152
-5867 -3482
-7533 9617
2171 -1330
3680 3759
5263 -4002
9120 -1346
1955 -6007
-2806 -8824
-1842 2251
-713 -5135
1862 7096
6943 -5039
2273 5046
-7818 7280
-480 -4625
4512 2861
-2432 7755
-4513 4227
-3302 -2987
-5323 1402
-666 3356
8670 -2762
5523 6118
-6366 -5717
6999 -2671
2199 -3728
7551 9300
9015 7582
-4670 3916
-2197 -5400
5792 -6293
3727 797
2739 -8510
-1042 6504
9492 -3905
-8569 -6284
-9208 8398
-7601 -7244
1581 2110
-6121 -3065
-1250 -9602
5763 -3409
-7461 -674
-9535 6392
8115 7847
5362 -6738
-9255 -7314
2878 -3357
-3392 -3065
5289 164
9331 9820
4469 7192
-3141 -3416
-9481 8179
2669 3784
1825 3158
6941 -3755
-9704 1846
374 -2944
8284 -4115
-5325 -8597
-821 4673
6974 9173
605 -7025
3896 -9137
-4733 6395
-9671 6326
9747 -5071
-7429 -1584
4187 3628
-8180 -3138
-8032 5138
8249 -6225
-9712 4124
7772 5345
-3290 3215
2253 121
-8390 2538
134 499
-5145 -7449
6814 6849
8490 -8882
1232 1353
8893 -2809
-4188 127
-1317 -7809
5990 -5871
5093 -2794
7569 -933
6188 -450
6307 -6506
3203 -2282
-4849 -2154
-4437 7599
2764 4637
-9442 5658
-6749 -4884
308 -6596
1480 8386
7407 -7023
1352 5979
-2104 5061
-2080 1248
1679 -9495
5308 -7320
353 -2110
-4982 -34
5292 -3758
-9472 9426
-9484 4170
-3910 3383
1723 -193
7128 -6968
-9908 -6712
-5408 2167
3855 -1059
739 -7702
-8257 8778
-7830 -282
-3663 -9873
-2603 665
-1752 8633
574 1557
784 -8301
1585 -3528
2303 5443
7846 998
991 6392
-3056 9341
7916 1943
-1112 -8198
3689 -3402
-5944 8398
-2325 347
-7514 855
1450 -8328
7448 -2826
7070 9078
-1387 -2692
7233 8347
6408 -8926
8098 -2352
8943 -4231
-3974 8089
-7313 658
3356 -6644
-3508 -725
-71 372
923 545
1256 -1524
-800 -3893
6516 -6164
4362 837
-3819 -7273
-2322 -2816
-8661 3832
-64 4890
-2663 -3585
5587 8462
-3836 5141
-8444 -2356
-1235 1013
-951 -3517
-6293 -1208
-3442 9827
-4764 -1920
-5691 8282
4518 -8995
8354 6128
4335 -7701
5242 162
-754 9831
4031 8732
-487 -1812
8233 -7441
-6049 590
1432 -4352
9654 833
-5876 5904
-3535 1
1231 -9896
-6251 -2765
-8848 -5421
-1870 -261
-1598 -8485
-8872 -6679
2916 -6100
-9410 4343
-4787 -8924
-8885 5106
9423 -2019
-8011 -7071
-9090 -5546
9230 1028
9867 -8399
4733 -149
9473 6058
-6989 8142
5803 -116
5785 6824
6330 -7990
7537 4632
-5022 5267
-2783 -5977
-4639 -5902
-1518 4489
6760 -4446
4071 -5866
-9479 -8814
8594 -7874
-6996 4651
5587 7366
-6072 -5594
-4054 -5949
682 -641
1932 2026
-4265 -9254
-1543 9821
2416 1623
3655 2707
-5651 -7339
-1508 -5278
-1555 -7671
-788 7432
1634 5080
-712 -994
4501 9453
7246 1007
-4199 7989
-9911 -5312
2037 -4968
3190 -9452
-747 1068
-9615 -4220
-3843 -8547
-8348 -4408
5791 -8401
17 -9600
-5992 -2985
9972 -4356
-4228 -9392
218 -9697
4346 4280
6020 -8857
-2414 618
-2388 6722
-5144 3439
5751 4348
4888 8989
-3790 9797
-8866 4480
453 -8278
5458 9061
863 -9033
24 -9194
-496 -9172
8000 -7230
-3752 -4465
-9955 -3401
-4583 -8897
-4869 333
2985 -9699
-964 3046
9431 -4342
5663 -896
-7059 8954
6595 -937
-3139 -1510
-64 2950
8604 5827
-6545 1721
-4158 -237
-4883 6658
-4379 8194
3881 6239
-5777 8744
7464 7580
3786 -4529
-1219 6833
3190 5721
4893 5709
6707 -6851
5873 -2186
1700 -4489
6470 9415
-7888 -6117
9375 9811
663 -9506
2897 7532
1689 7096
7324 -4618
-2721 -459
9186 -4285
-2206 -2194
3807 6929
-4833 -9582
-36 -5594
-2964 -8811
3343 3054
2503 4577
601 9785
3580 8347
-3386 999
-2708 1030
8125 9914
-9452 7953
7921 7048
-9054 9215
-9049 -7465
-8520 1838
-7716 6112
1181 -7143
6496 -3924
-9325 -5886
7435 559
3308 -3047
1365 2067
