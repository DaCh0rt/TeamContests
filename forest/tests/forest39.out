82818.57 457627800.07
